internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -492,7 +512,8 @@ public RssSendShuffleDataResponse sendShuffleData(RssSendShuffleDataRequest requ
                       partitionIds,
                       allocateSize,
                       request.getRetryMax() / maxRetryAttempts,
-                      request.getRetryIntervalMax());
+                      request.getRetryIntervalMax(),
+                      failedStatusCode);
```

Comment:
`ShuffleServerGrpcNettyClient#sendShuffleData` also seems to require `failedStatusCode`

Reply:
yes. I forgot this part.

Reply:
Done